app/scripts/controllers/TasksCtrl.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+   function TasksCtrl(Tasks, $scope) {
+     $scope.allTasks = Tasks.all;
+
+     $scope.addTask = function() {
```

Comment:
@melissajstudent it might be a little better organization to store this function in the `Task` service rather than in the controller. That way theoretically if you had a more complex app you'd be able to add tasks from anywhere. No need to update it for this project, but just general organizational feedback.